scripts/README.md

```diff
@@ -6,6 +6,10 @@ The files in this folder were utilized to create or adjust the `scpca-meta.json`
 In this script, the old results from mapping are copied over from the `internal` to `checkpoints` directory.
 Additionally, the `scpca-meta.json` file was created and saved to the `checkpoints` directory for each run to track processing information.
 
-2. `add-fields-scpca-meta.py`: This script specifically updates existing `scpca-meta.json` files within the `checkpoints` directory.
+2. `add-fields-scpca-meta.py`: This script specifically updates existing `scpca-meta.json` files within the `checkpoints` directory for mapping steps.
 For any libraries processed through `scpca-nf v0.5.1` or earlier, the `scpca-meta.json` file is modified to include additional fields that have been added in new release of `scpca-nf`.
 This includes `ref_mito`, `ref_fasta_index`, `assay_ontology_term_id`, and `submitter_cell_types_file`.
+
+3. `add-celltype-fields-scpca-meta.py`: This script specifically updates existing `scpca-meta.json` files in the `checkpoints/celltype` directory.
+For any libraries that have been processed through `scpca-nf v0.6.2` or earlier, the `scpca-meta.json` files created for cell type annotation are modified to include the most up-to-date reference file names.
```

Comment:
I just want to rephrase this to confirm it will only update files names for names that were used before the current naming scheme. We don't update if the reference file was updated and an older reference was used.